src/Controls/src/Core/Compatibility/Handlers/TableView/Android/TableViewRenderer.cs

```diff
@@ -107,7 +137,10 @@ public override SizeRequest GetDesiredSize(double widthConstraint, double height
 							continue;
 						}
 
-						AView listItem = _adapter.GetView(i, null, Control);
+						var platformView = cell.Handler?.PlatformView as AView;
+						var convertView = (platformView?.Parent as AView) ?? platformView;
```

Comment:
How do we know parent is the right view to use?

Reply:
I'll add a comment